gamatrix-gog.py

```diff
@@ -71,6 +71,10 @@ def compare_libraries():
         igdb.get_game_info(release_key)
         igdb.get_multiplayer_info(release_key)
 
+    # Save the cache to disk if we added anything
+    if cache.data.dirty:
```

Comment:
Consider adding function/property `Cache.is_dirty` and checking the `CacheDict.dirty` from within that module - then no-one else needs to know about your internal implementation of the cache.